src/notifications/tests/notification.test.ts

```diff
@@ -0,0 +1,76 @@
+import { DateTime } from "luxon";
+import { UNSOCIAL_HOUR_END, UNSOCIAL_HOUR_START } from "../../app/constants";
+import { handleOngoingCleaningStateUpdate } from "../helper-functions/handleOngoingCleaningStateUpdate";
+import { isUnsocialHour } from "../helper-functions/unsocialHoursCalculationHelpers";
+
+
+describe("Tests unsocialhours definition", () => {
+  it("should return true for unsocial hours", () => {
+    expect(isUnsocialHour(5)).toBe(true);
+  });
+
+  it("should return true for unsocial hours", () => {
+    expect(isUnsocialHour(22)).toBe(true);
+  });
+
+  it("should return false for unsocial hours", () => {
+    expect(isUnsocialHour(7)).toBe(false);
+  });
+
+  it("should return false for UNSOCIAL_HOUR_END", () => {
+    expect(isUnsocialHour(UNSOCIAL_HOUR_END)).toBe(false); // 6am should be social
+  });
+  it("should return true for UNSOCIAL_HOUR_START", () => {
+    expect(isUnsocialHour(UNSOCIAL_HOUR_START)).toBe(true); // 10pm should be unsocial
+  });
+})
+
+
```

Comment:
I now understand the value of using reducers, as this is really tricky to test! I think is a valid solution, but it is new to me!